### PR TITLE
[nearlib] config: read contract api url from passed-in config too

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: nearlib/dev.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ module.exports = {
         fullRuntimeOptions.nodeUrl = fullRuntimeOptions.nodeUrl || (await this.getConfig()).nodeUrl || localNodeUrl;
         fullRuntimeOptions.deps.keyStore = fullRuntimeOptions.deps.keyStore || new nearlib.BrowserLocalStorageKeystore(fullRuntimeOptions.networkId),
         fullRuntimeOptions.deps.storage = fullRuntimeOptions.deps.storage || window.localStorage;
+        fullRuntimeOptions.baseUrl = fullRuntimeOptions.baseUrl || (await this.getConfig()).baseUrl;
```
Comment (vgrichina) [inline nearlib/dev.js]: nit: store result of `await this.getConfig()` (as it potentially involves I/O)

File: nearlib/dev.js [hunk 2]
```diff
@@ -88,7 +89,7 @@ module.exports = {
      * Function to create an account on near-hosted devnet using contract helper. This will not work on non-dev environments.
      */
     createAccountWithContractHelper: async function (nearConfig, newAccountId, publicKey) {
-        return await sendJson('POST', `${nearConfig.baseUrl}/account`, {
+        return await sendJson('POST', `${this.config.baseUrl}/account`, {
```
Comment (vgrichina) [inline nearlib/dev.js]: shouldn't take `nearConfig` as argument anymore then?

**Comment by vgrichina** [comment]
Do you plan to merge this?

**Comment by janedegtiareva** [comment]
I think this is obsolete now

**Pull request closed without merging**